### DISCUSSION TITLE
Will there by any possabilities MOJO could be used for web development in the future?

### DISCUSSION THREAD

**Discussion by Cafezinho** [modular/modular#465] [General]
Will there by any possabilities MOJO could be used for web development in the future?

**Comment by Mogball** [comment]
It's possible. Many have asked for a webassembly target, which is possible through LLVM.

**Reply by Bokomoko** [reply]
That would be great. Is it being implemented with Mojo already?

**Comment by lattner** [comment]
This isn't a current priority for us.  I'd recommend checking out the mojodojo site which has collected answers to many common questions:
https://mojodojo.dev/mojo-team-answers.html#wasm

**Comment by Huhan001** [comment]
would love to see this.
